src/tableDrawer.ts

```diff
@@ -392,9 +392,12 @@ function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
       doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
     }
   } else if (typeof cellStyles.lineWidth === 'object') {
-    doc.rect(cell.x, cursor.y, cell.width, cell.height, 'F')
 
     const sides = Object.keys(cellStyles.lineWidth)
+    if(sides.length === 4) {
+      const fillColor = cellStyles.fillColor === false ? null : typeof cellStyles.fillColor !== "string" ? 'F' : cellStyles.fillColor;
```

Comment:
`false` should map to null
non strings to 'F'
strings can be passed to `jsPdf.rect`